src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/TimePeriodRangeViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Public.Data;
+
+/// <summary>
+/// Describes a time period range in human-readable format.
+/// </summary>
```

Comment:
Similar to `DataSetViewModels`, do we want to keep these kind of self-explanatory docblocks?